common/workspace-hack/Cargo.toml

```diff
@@ -191,6 +207,9 @@ codespan-reporting = { version = "0.8.0", default-features = false, features = [
 crossbeam-channel = { version = "0.5.1", features = ["crossbeam-utils", "default", "std"] }
 crossbeam-deque = { version = "0.8.0", features = ["crossbeam-epoch", "crossbeam-utils", "default", "std"] }
 crossbeam-utils = { version = "0.8.3", features = ["default", "lazy_static", "std"] }
+curve25519-dalek-fiat = { version = "0.1.0", default-features = false, features = ["alloc", "fiat-crypto", "fiat_u64_backend", "std"] }
+ed25519 = { version = "1.0.3", default-features = false, features = ["serde", "std"] }
```

Comment:
can you try removing `ed25519` deps? we already have `ed25519-dalek-fiat` which is a fork of `ed25519`

Reply:
Let me try this. Also, this file is auto-generated via `cargo x generate-workspace-hack`, so I guess if removing `ed25519` works, we might need to change the `x` tool a bit as well.

Reply:
Removing `ed25519` does not seem to be working but I have another idea as I noticed the `hakari` section in `x.toml`:

```
[[hakari.omitted-packages]]
name = "diem-crypto"
version = "0.1.0"
workspace-path = "crypto/crypto"
```

The idea is to isolate the crypto part of the stackless VM and put it into a thin crate (maybe called `bytecode-interpreter-crypto` etc) and apply the samek `hakari` annotation to that package.

@vgao1996 let me know if there will be other use cases of the `diem-crypto` repo in Move crates. Because if there are other dependencies to `diem-crypto`, we might want to make the `bytecode-interpreter-crypto` more general.

Reply:
There aren't other Move crates that depend on Move crypto. The last one was removed in #8566.

Reply:
Cool, then I'll just confine the thin layer within the interpreter folder structure-wise.